components/hab/src/cli.rs

```diff
@@ -737,15 +737,25 @@ fn sub_pkg_install() -> App<'static, 'static> {
         (@arg FORCE: -f --force "Overwrite existing binlinks")
         (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
     );
-    if feat::is_enabled(feat::OfflineInstall) {
+    sub = if feat::is_enabled(feat::OfflineInstall) {
         sub.arg(
             Arg::with_name("OFFLINE")
                 .help("Install packages in offline mode")
                 .long("offline"),
         )
     } else {
         sub
-    }
+    };
+    sub = if feat::is_enabled(feat::IgnoreLocal) {
+        sub.arg(
+            Arg::with_name("IGNORE_LOCAL")
+                .help("Do not use locally-installed packages when a corresponding package cannot be installed from Builder")
+                .long("ignore-local"),
+        )
+    } else {
+        sub
+    };
```

Comment:
Just out of curiosity is there a reason why
```rust
    sub = if feat::is_enabled(feat::OfflineInstall) {
        sub.arg(
            …
        )
    } else {
        sub
    };
```
rather than
```rust
    if feat::is_enabled(feat::IgnoreLocal) {
        sub = sub.arg(
            …
        )
    }
```
?

I understand if you're maintaining consistency with the existing pattern above, but I wanted to make sure there wasn't a deeper reason like a rust thing that I wasn't aware of.

Reply:
Nah, no reason... your way is cleaner; will change.